src/bioregistry/data/bioregistry.json

```diff
@@ -15171,6 +15398,11 @@
     "biocontext": {
       "prefix": "CHEMIDPLUS"
     },
+    "contact": {
+      "name": "Patricia Tomasulo",
+      "email": "tomasulo@library.med.nyu.edu",
+      "comment": "Old email found no new email findable. No Id on .doesnt work there anymore."
+    },
```

Comment:
If the email is not active anymore, it's better if we don't list it, I suggest removing